bodhi/tests/server/tasks/test_composer.py

```diff
@@ -540,7 +540,8 @@ def test_tag_ordering(self, *args):
 
         with self.db_factory() as session:
             firstupdate = session.query(Update).one()
-            build = RpmBuild(nvr=otherbuild, package=firstupdate.builds[0].package, signed=True)
+            build = RpmBuild(nvr=otherbuild, package=firstupdate.builds[0].package,
+                             release=firstupdate.release, signed=True)
```

Comment:
why do we need to pass the release now ?

Reply:
Because of the change in `models.RpmBuild.get_latest()`:
```
for tag in [self.update.release.stable_tag, self.update.release.dist_tag]:
```
has been changed to 
```
for tag in [self.release.stable_tag, self.release.dist_tag]:
```
and a Build object is supposed to always have a Release associated. Previously the release was acquired from the update by linking Build -> Update -> Release, now it is acquired directly from Build -> Release and the mocked Build here in the tests was not happy with that.